### ISSUE TITLE
can't edit podcast info

### ISSUE THREAD

**Issue by waxweb** [PodcastGenerator/PodcastGenerator#401]
**System Information**
Podcast Generator Version: 
Webserver: 
PHP Version:

**Podcast Generator Config**

```
INSERT THE **CENSORED** CONFIG HERE
``` 

**Description of your problem:**

hi I have a very simple problem.... I can't edit podcast details, or change the cover picture. Where are the permissions needed to change this? I have an automatic install on Bluehost. Dirs are set to chmod 0755, files to 0644. I don't see anything that would help me in the config file itself. Thanks in advance! :)

**Comment by emilengler** [comment]
Can you change dirs to 775 and files to 664?

**Comment by waxweb** [comment]
yes, dirs are 775, files are 664. I was able to ftp-index a file I uploaded to /media, and I see in the created xml that the uneditable references are internal to the script

**Comment by emilengler** [comment]
> yes, dirs are 775, files are 664. I was able to ftp-index a file I uploaded to /media, and I see in the created xml that the uneditable references are internal to the script

Are you sure? Your initial description said something contradictory.
